src/node/nodeValues.ts

```diff
@@ -66,11 +66,16 @@ const getValuePropRaw = (params: { node: Node; prop: string; defaultValue?: any
 const getItemUuid = (node: Node): string | undefined =>
   getValuePropRaw({ node, prop: valuePropsCode[valuePropsCode.itemUuid] })
 
+const getDateTimePart = (params: { node: Node; index: number; defaultValue: string; separator: string }): number => {
```

Comment:
defaultValue could be defaultSeparator, what do u think?

Reply:
uhm, that default value is actually the separator repeated 1 time for the time value and 2 times for the date value... but actually we don't even need a default value. we can do this:
Number((node.value || '').split(separator)[index]?.trim())

Reply:
better I felt that defaultValue is the defaultValue of the DateTimePart as the name of the function is getDateTimePart ajaja